<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="image_backdop">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>wp3790948-anime-aesthetic-wallpapers_900x600.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pomodoro">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color: rgba(0,0,0,0);
color: rgb(255, 255, 255);
border-radius: 20px;
border: 0.5px solid white;}

QPushButton:hover {
              color: rgb(0,0,0);
		     background-color: rgb(225,225,225);
			
		
}
QPushButton:pressed {
							
	font: 11pt &quot;Consolas&quot;;
}</string>
   </property>
   <property name="text">
    <string>pomodoro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

		background-color: rgb(255, 255, 255);
		border-radius: 20px;
		border: 0.5px solid white;

}
QPushButton:hover {
								border-radius: 20px;		
								border: 0.5px solid white;
								background-color: rgb(0, 0, 0);
								color: white;
								
}
QPushButton:pressed {
							
								
	font: 11pt &quot;Consolas&quot;;
}</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="reset">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>400</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">


QPushButton{

background-color: rgba(255, 255, 255, 0);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons8-restart-58.png</normaloff>icons8-restart-58.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="second">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>170</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="minute">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>60</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="break5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">.QPushButton{background-color: rgba(0,0,0,0);
					color: rgb(255, 255, 255);
					border-radius: 20px;
					border: 0.5px solid white;
   }

QPushButton:hover {
        color: rgb(0,0,0);
		background-color: rgb(225,225,225);
		
}
QPushButton:pressed {
							
	font: 11pt &quot;PMingLiU-ExtB&quot;;
}</string>
   </property>
   <property name="text">
    <string>break</string>
   </property>
  </widget>
  <widget class="QLabel" name="msg">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>481</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>!Select a mode!</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="stop">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>120</width>
     <height>55</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{

		background-color: rgb(255, 255, 255);
		border-radius: 20px;
		border: 0.5px solid white;

}
QPushButton:hover {
								border-radius: 20px;		
								border: 0.5px solid white;
								background-color: rgb(0, 0, 0);
								color: white;
								
}
QPushButton:pressed {
							
								
	font: 11pt &quot;Consolas&quot;;
}</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="C:/Users/boomh/Downloads/resource.qrc"/>
 </resources>
 <connections/>
</ui>
